cypress/cypress/integration/video.spec.ts

```diff
@@ -0,0 +1,168 @@
+    mockDefaultSetup(cy, {
+      config: { mentorsDefault: ["clint", "carlos"] },
+      mentorData: [clint, carlos],
+      apiResponse: "response_with_feedback.json",
+      gqlQueries: [
+        cyMockGQL("UserQuestionSetFeedback", { userQuestionSetFeedback: null }),
+      ],
+    });
+    cy.intercept("**/questions/?mentor=clint&query=*", {
+      fixture: "response_with_feedback.json",
+    });
+    cy.visit("/");
+
+    // Mentor Card in the beggining
+    cy.get("[data-cy=video-container").within(($hc) => {
+      cy.get("[data-cy=mentor-name-card]").should("exist");
+      cy.get("[data-cy=mentorname-faveicon-wrapper]").within(($hc) => {
+        cy.get("[data-cy=mentor-name]")
+          .should("exist")
+          .contains("Clinton Anderson");
+        cy.get("[data-cy=fave-button]").should("exist");
+      });
+    });
+
+    cy.get("[data-cy=header]").should("have.attr", "data-mentor", "clint");
+    cy.get("[data-cy=topic-2] button").trigger("mouseover").click();
+    cy.get("[data-cy=history-chat]").should("exist");
+
+    cy.get("[data-cy=input-field]").type("user msg 1");
+    cy.get("[data-cy=input-send]").trigger("mouseover").click();
+
+    cy.get("[data-cy=history-chat").within(($hc) => {
+      cy.get("[data-cy=chat-msg-1]").contains("user msg 1");
+    });
+
+    // Mentor Card at the end
+    cy.get("[data-cy=video-container").within(($hc) => {
+      cy.get("[data-cy=mentor-name-card]").should("exist");
+      cy.get("[data-cy=mentorname-faveicon-wrapper]").within(($hc) => {
+        cy.get("[data-cy=mentor-name]")
+          .should("exist")
+          .contains("Clinton Anderson");
+        cy.get("[data-cy=fave-button]").should("exist");
+      });
+    });
+  });
+
+  it("Title Header for one mentor", () => {
+    mockDefaultSetup(cy, {
+      config: {
+        cmi5Enabled: false,
+        mentorsDefault: ["clint"],
+        styleHeaderColor: "#990000",
+        styleHeaderTextColor: "#FFFFFF",
+        styleHeaderLogo:
+          "http://scribe.usc.edu/wp-content/uploads/2021/02/PrimShield_Word_SmUse_Gold-Wh_RGB-1.png",
+      },
+      mentorData: [clint],
+    });
+    cy.viewport(1200, 700);
+    cy.intercept(
+      "http://scribe.usc.edu/wp-content/uploads/2021/02/PrimShield_Word_SmUse_Gold-Wh_RGB-1.png",
+      { fixture: "uscheader2.png" }
+    );
+    cy.intercept("**/questions/?mentor=clint&query=*", {
+      fixture: "response_with_feedback.json",
+    });
+    cy.visit("/");
+    cy.get("[data-cy=header] img")
+      .should("have.attr", "src")
+      .and(
+        "eq",
+        "http://scribe.usc.edu/wp-content/uploads/2021/02/PrimShield_Word_SmUse_Gold-Wh_RGB-1.png"
+      );
+    cy.get("[data-cy=header]").should(
+      "have.css",
+      "background-color",
+      "rgb(153, 0, 0)"
+    ); //RGB of #990000
+    cy.get("[data-cy=header] p").should(
+      "have.css",
+      "color",
+      "rgb(255, 255, 255)"
+    ); //RGB of #FFFFFF
+
+    cy.get("[data-cy=header]").contains(
```

Comment:
this is an `.and` for the `within` above

Reply:
OK